ngraph/python/src/ngraph/opset6/ops.py

```diff
@@ -58,6 +58,35 @@
 # -------------------------------------------- ops ------------------------------------------------
 
 
+@nameable_op
+def ctc_greedy_decoder_seq_len(
+        data: NodeInput,
+        sequence_length: NodeInput,
+        blank_index: Optional[int] = 0,
```

Comment:
As far as I understand default value for `blank_index` is equal to `num_classes - 1`. Is it set to this value while processing in cpp internals? If not then I think you should change 0 to `None`, put an if statement to get length of the 3rd dimension from `data` and subtract one.